<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualize</class>
 <widget class="QDialog" name="Visualize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>565</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLCDNumber" name="cell1_2">
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Filled</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="cell2_2">
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Filled</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="cell3_2">
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="cell4">
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="cell5">
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="cell6">
        <property name="digitCount">
         <number>3</number>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="cell7">
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="cell8">
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="Slider">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>180</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="Box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>563</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Speed control</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
